entrypoint.sh

```diff
@@ -8,13 +8,11 @@ if [[ -z "${SONAR_TOKEN}" ]]; then
 fi
 
 if [[ -f "${INPUT_PROJECTBASEDIR%/}/pom.xml" ]]; then
-  echo "Maven project detected. You should run the goal 'org.sonarsource.scanner.maven:sonar' during build rather than using this GitHub Action."
-  exit 1
+  echo "WARNING! Maven project detected, you might have only partial analysis. You can run the goal 'org.sonarsource.scanner.maven:sonar' during build instead of this GitHub Action."
```

Comment:
```suggestion
  echo "WARNING! Maven project detected, you might have only partial analysis. You can run the goal 'org.sonarsource.scanner.maven:sonar-maven-plugin:sonar' during build instead of this GitHub Action."
```